main.tf

```diff
@@ -546,6 +546,416 @@ module "external_secrets" {
+  namespace        = try(var.aws_load_balancer_controller.namespace, "kube-system")
+  # namespace creation is false here as kube-system already exists by default
+  create_namespace = try(var.aws_load_balancer_controller.create_namespace, false)
+  chart            = local.aws_load_balancer_controller_name
+  chart_version    = try(var.aws_load_balancer_controller.chart_version, "1.4.8")
+  repository       = try(var.aws_load_balancer_controller.repository, "https://aws.github.io/eks-charts")
+  values           = try(var.aws_load_balancer_controller.values, [])
+
+  timeout                    = try(var.aws_load_balancer_controller.timeout, null)
+  repository_key_file        = try(var.aws_load_balancer_controller.repository_key_file, null)
+  repository_cert_file       = try(var.aws_load_balancer_controller.repository_cert_file, null)
+  repository_ca_file         = try(var.aws_load_balancer_controller.repository_ca_file, null)
+  repository_username        = try(var.aws_load_balancer_controller.repository_username, null)
+  repository_password        = try(var.aws_load_balancer_controller.repository_password, null)
+  devel                      = try(var.aws_load_balancer_controller.devel, null)
+  verify                     = try(var.aws_load_balancer_controller.verify, null)
+  keyring                    = try(var.aws_load_balancer_controller.keyring, null)
+  disable_webhooks           = try(var.aws_load_balancer_controller.disable_webhooks, null)
+  reuse_values               = try(var.aws_load_balancer_controller.reuse_values, null)
+  reset_values               = try(var.aws_load_balancer_controller.reset_values, null)
+  force_update               = try(var.aws_load_balancer_controller.force_update, null)
+  recreate_pods              = try(var.aws_load_balancer_controller.recreate_pods, null)
+  cleanup_on_fail            = try(var.aws_load_balancer_controller.cleanup_on_fail, null)
+  max_history                = try(var.aws_load_balancer_controller.max_history, null)
+  atomic                     = try(var.aws_load_balancer_controller.atomic, null)
+  skip_crds                  = try(var.aws_load_balancer_controller.skip_crds, null)
+  render_subchart_notes      = try(var.aws_load_balancer_controller.render_subchart_notes, null)
+  disable_openapi_validation = try(var.aws_load_balancer_controller.disable_openapi_validation, null)
+  wait                       = try(var.aws_load_balancer_controller.wait, null)
+  wait_for_jobs              = try(var.aws_load_balancer_controller.wait_for_jobs, null)
+  dependency_update          = try(var.aws_load_balancer_controller.dependency_update, null)
+  replace                    = try(var.aws_load_balancer_controller.replace, null)
+  lint                       = try(var.aws_load_balancer_controller.lint, null)
+
+  postrender = try(var.aws_load_balancer_controller.postrender, [])
+  set = concat([
+    {
+      name  = "clusterName"
```

Comment:
we can remove this first entry for clusterName since its not used by ALB controller